### PR TITLE
add benchmark for animated complex opacity

### PR DESCRIPTION
**Pull request by jonahwilliams**
benchmark to show improvement in https://github.com/flutter/flutter/pull/101952 towards addressing https://github.com/flutter/flutter/issues/101597 . Running locally shows about a 50% raster time improvement

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -1453,6 +1453,17 @@ targets:
       task_name: animated_image_gc_perf
     scheduler: luci
 
+  - name: Linux_android animated_complex_opacity_perf__e2e_summary
```
Comment (keyonghan) [inline .ci.yaml:1456]: We need an ownership entry in [animated_complex_opacity_perf__e2e_summary](https://github.com/flutter/flutter/blob/master/TESTOWNERS#L11)

Comment (jonahwilliams) [reply .ci.yaml:1456]: Done

Comment (keyonghan) [reply .ci.yaml:1456]: Uh, the suffix `.dart` is also needed to bypass the check.

Comment (jonahwilliams) [reply .ci.yaml:1456]: Okay now done :)

**Review by keyonghan — APPROVED** [review]
The configs LGTM.

File: dev/benchmarks/macrobenchmarks/android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -21,6 +21,7 @@ found in the LICENSE file. -->
         android:label="macrobenchmarks"
         android:icon="@mipmap/ic_launcher">
         <activity
+            android:exported="true"
```
Comment (flar) [inline dev/benchmarks/macrobenchmarks/android/app/src/main/AndroidManifest.xml:24]: I see this line popping up when developers create a new benchmark. Is it needed? Is it an issue? Should I just let it go through?

**Review by flar — APPROVED** [review]
LGTM, modulo question about that new flag in the build files...

File: dev/benchmarks/macrobenchmarks/android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (jonahwilliams) [reply dev/benchmarks/macrobenchmarks/android/app/src/main/AndroidManifest.xml:24]: This is a config change needed to run on the newest version of android. Adding an automatic migration is tracked in https://github.com/flutter/flutter/issues/102183

**Pull request merged**